### ISSUE TITLE
Problem with convert xunit tests with different InlineData

### ISSUE THREAD

**Issue by szczepix** [gabrielweyer/xunit-to-junit#3]
# Problem

Wrong name (_the same name_) when the unit test has a lot of `InlineData`. The name should contain information about the inline data. Important: [InlineData](http://dontcodetired.com/blog/post/Creating-Inline-Data-Driven-Tests-in-xUnit) contains [polish characters](https://en.wikipedia.org/wiki/Polish_alphabet) with [diacritics](https://en.wikipedia.org/wiki/Diacritic).

### Input:

```xml
<!-- ... -->
<test name="UnitTestsProject.PolishCharacterMapperTests.Schould_Map2Polish_returns_valid_mapped_string(input: \&quot;A\&quot;, output: \&quot;[AaĄą]\&quot;)" type="UnitTestsProject.PolishCharacterMapperTests" method="Schould_Map2Polish_returns_valid_mapped_string" time="0.0000037" result="Pass" />
<test name="UnitTestsProject.PolishCharacterMapperTests.Schould_Map2Polish_returns_valid_mapped_string(input: \&quot;ń\&quot;, output: \&quot;[Ńń]\&quot;)" type="UnitTestsProject.PolishCharacterMapperTests" method="Schould_Map2Polish_returns_valid_mapped_string" time="0.0000042" result="Pass" />
<!-- ... -->
```

### Output: (actual -WRONG)

```xml
<!-- ... -->
<testcase name="Schould_Map2Polish_returns_valid_mapped_string" classname="UnitTestsProject.PolishCharacterMapperTests" time="0.0000037" />
<testcase name="Schould_Map2Polish_returns_valid_mapped_string" classname="UnitTestsProject.PolishCharacterMapperTests" time="0.0000042" />
<!-- ... -->
```

### Output:

```xml
<!-- ... -->
<testcase name="Schould_Map2Polish_returns_valid_mapped_string(input: \&quot;ń\&quot;, output: \&quot;[Ńń]\&quot;)" classname="UnitTestsProject.PolishCharacterMapperTests" time="0.0000037" />
<testcase name="Schould_Map2Polish_returns_valid_mapped_string(input: \&quot;ń\&quot;, output: \&quot;[Ńń]\&quot;)" classname="UnitTestsProject.PolishCharacterMapperTests" time="0.0000042" />
<!-- ... -->
```

**Comment by gabrielweyer** [comment]
Thanks for the clear bug report. I didn't consider `InlineData` support initially as I only used the converter for some basic tests. I've now added support for `InlineData` and published [v2.0.0](https://www.nuget.org/packages/dotnet-xunit-to-junit/2.0.0). Please let me know if you experience any issue.

**Comment by szczepix** [comment]
Thank you so much 🤝

It works as intended:

![image](https://user-images.githubusercontent.com/1363477/91199228-e4534c00-e6fd-11ea-9ef9-273fb6d3e4de.png)
